app/src/main/java/org/keepgoeat/presentation/withdraw/WithdrawReasonAdapter.kt

```diff
@@ -0,0 +1,64 @@
+package org.keepgoeat.presentation.withdraw
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import org.keepgoeat.R
+import org.keepgoeat.databinding.ItemWithdrawBinding
+import org.keepgoeat.presentation.model.WithdrawReason
+import org.keepgoeat.presentation.type.WithdrawCheckType
+import org.keepgoeat.util.ItemDiffCallback
+
+class WithdrawReasonAdapter(context: Context) :
+    ListAdapter<WithdrawReason, RecyclerView.ViewHolder>(
```

Comment:
```suggestion
class WithdrawReasonAdapter(context: Context) :
    ListAdapter<WithdrawReason, WithdrawReasonAdapter.WithdrawViewHolder>(
```

해당 어뎁터는 멀티뷰홀더를 사용할 여지는 없어보입니다~ 요렇게 바꿔주세용!